src/main/java/chacko/ben/redis/RedisInfoController.java

```diff
@@ -92,25 +99,43 @@ void killServer() {
         System.exit(0);
     }
 
-    private Jedis getJedisConnection() {
+    private void getJedisConnection() {
         // get our connection info from VCAP_SERVICES
         RedisInstanceInfo info = getInfo();
-        Jedis jedis = new Jedis(info.getHost(), info.getPort());
+        pool = new JedisPool(new JedisPoolConfig(), info.getHost(), info.getPort(), Protocol.DEFAULT_TIMEOUT, info.getPassword());
         
-        // make the connection
-        jedis.connect();
-
-        // authorize with our password
-        jedis.auth(info.getPassword());
-
-        return jedis;
+//        Jedis jedis = new Jedis(info.getHost(), info.getPort());
+//        
+//        // make the connection
+//        jedis.connect();
+//
+//        // authorize with our password
+//        jedis.auth(info.getPassword());
+        
+        //return pool;
     }
     
     @RequestMapping("/start/{key}")
     void startLoadTest(@PathVariable(value="key") String key) throws InterruptedException {
     	_continue = true;
+    	pool.getResource();
     	while(_continue) {
-    		String value = getKey(key);
+    		String value = "";
+    		try
+            {
+    			jedis = pool.getResource();
+                value = jedis.get(key);
+            }
+            catch (JedisConnectionException e)
+            {
+                
+            	LOG.log(Level.WARNING, "About to kill the service!");
+            }
+            finally
+            {
+                jedis.close();
+            }
+    		//String value = getKey(key);
```

Comment:
Can you run the formatter on the whole file so that the code is lined up properly: (select all text) -> command-f